release-notes/10.0/preview/preview6/aspnetcore.md

```diff
@@ -0,0 +1,239 @@
+
+**How to use:**
+No action is needed to benefit from this feature. Memory eviction is handled automatically by the framework.
+
+There are also metrics added to the default memory pool used by our server implementations. The new metrics are under the name `Microsoft.AspNetCore.MemoryPool`. See the [ASP.NET Core metrics documentation](https://learn.microsoft.com/aspnet/core/log-mon/metrics/metrics) for general information on what metrics are and how to use them.
+
+You can also use the new `IMemoryPoolFactory` service interface to create or access memory pools for custom scenarios:
+
+```csharp
+public class MyBackgroundService : BackgroundService
+{
+    private readonly MemoryPool<byte> _memoryPool;
+
+    public MyBackgroundService(IMemoryPoolFactory<byte> factory)
+    {
+        _memoryPool = factory.Create();
+    }
+
+    protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+    {
+        while (!stoppingToken.IsCancellationRequested)
+        {
+            try
+            {
+                await Task.Delay(20, stoppingToken);
+                // do work that needs memory
+                var rented = _memoryPool.Rent(100);
+                rented.Dispose();
+            }
+            catch (OperationCanceledException)
+            {
+                return;
+            }
+        }
+    }
+}
+```
+
+Use a custom `IMemoryPoolFactory` to replace the memory pool being used:
+
+```csharp
+services.AddSingleton<IMemoryPoolFactory<byte>, CustomMemoryPoolFactory>();
+
+public class CustomMemoryPoolFactory : IMemoryPoolFactory<byte>
+{
+    public MemoryPool<byte> Create()
+    {
+        // Return a custom MemoryPool implementation or the default.
+        return MemoryPool<byte>.Shared;
+    }
+}
+```
+
+## Blazor WebAssembly preloading
+
+Blazor now provides a `<LinkPreload />` component to generate `link` tags instead of using link headers to preload framework assets. This allows the framework to correctly identify the Blazor application base URL and provides better control over the preloading behavior.
+
+To use this feature, place the `<LinkPreload />` component in the head of your application:
+
+```diff
+<head>
+  <base href="/" />
++  <LinkPreload />
+  ...
+</head>
+```
+
+Removing the component will disable the preloading feature, which is useful in cases where the application uses the `loadBootResource` callback to modify URLs.
+
+## Blazor build producing javascript bundler friendly output
+
+The Blazor build output is now compatible with tools like webpack or rollup when you set the `WasmBundlerFriendlyBootConfig` MSBuild property to `true`. This output is not directly runnable in the browser, but it can be consumed by JavaScript bundlers to combine with other user scripts.
```

Comment:
```suggestion
Set the new `WasmBundlerFriendlyBootConfig` MSBuild property to `true` to make the Blazor WebAssembly build output compatible with tools like webpack or rollup. Setting this property will adjust the .NET boot configuration to explicitly import framework assets. This output is not directly runnable in the browser, but it can be consumed by JavaScript bundlers to combine it with other scripts.
```